src/lib/annotations/Annotator.js

```diff
@@ -244,8 +244,8 @@ class Annotator extends EventEmitter {
      * @return {void}
      */
     hideAnnotations() {
-        Object.keys(this.threads).forEach((page) => {
-            this.threads[page].forEach((thread) => {
+        Object.values(this.threads).forEach((pageThreads) => {
```

Comment:
Can you follow the same idea for what I mention below, about `showAnnotationsOnPage()`